proposals/NNNN-rc-lights-more-names-and-status-values.md

```diff
@@ -0,0 +1,75 @@
+# Remote Control – Lights – More Names and Status Values
+
+* Proposal: [SDL-NNNN](NNNN-rc-lights-more-names-and-status-values.md)
+* Author: [Zhimin Yang](https://github.com/smartdevicelink/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC]
+
+## Introduction
+
+The light status signal in the vehicle can provide more values than just ON / OFF for certain lights. Some vehicles can provide the status of more lights than currently defined light names. This proposal adds three new light status values and some new light names to allow an application to get the status of additional lights.
+
+## Motivation
+
+The light status signal in the vehicle can provide more values than just ON / OFF for certain lights. 
+Some vehicles can provide the status of more lights than currently defined light names.
+Previously, we assume a RC application can always control the ON/OFF status of a light if the light name is listed in the `supportedLights`, which is in the `LightControlCapabilities`. However, a vehicle may not allow an application to control certain lights but provides the status of such lights. That means the status of those lights is read-only. The vehicle shall allow the application to know this fact via the capabilities.
```

Comment:
Please update the first sentence to "...we assumed..." and the second to "...lights but may provide the status..."

Reply:
updated